ui/mirage/handlers/sync.js

```diff
@@ -121,6 +121,13 @@ export default function (server) {
   server.post(`${uri}/associations/set`, (schema, req) => {
     const { type, name } = req.params;
     const { secret_name, mount } = JSON.parse(req.requestBody);
+    if (secret_name.slice(-1) === '/') {
+      return new Response(
+        400,
+        {},
+        { errors: ['Secret not found. Please provide full path to existing secret'] }
+      );
+    }
```

Comment:
Easy way to test error state in workflow by providing a secret path ending in a slash.